docs/content/docs/concepts/glossary.md

```diff
@@ -172,6 +172,10 @@ A Sub-Task is a [Task](#task) responsible for processing a [partition](#partitio
 the data stream. The term "Sub-Task" emphasizes that there are multiple parallel Tasks for the same
 [Operator](#operator) or [Operator Chain](#operator-chain).
 
+#### Table Program
```

Comment:
IMHO For each glossary entry there should always be a link to a page with more details about the topic, Perhaps this one https://ci.apache.org/projects/flink/flink-docs-release-1.13/docs/dev/table/common/ for this specific entry?